.github/jobs/configure-checks/all.bats

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/env bats
+
+load 'assert'
+
+u="domjudge-bats-user"
+
+setup_user() {
+    id -u $u || useradd $u >/dev/null
+    chown -R $u:$u ./
+}
+
+setup() {
+    setup_user
+    for shared_file in config.log confdefs.h conftest.err; do
+        chmod a+rw $shared_file || true
+    done
+    repo-install gcc g++ libcgroup-dev
+}
+
+q_configure () {
+    su $u -c "./configure $@"
+    #./configure $@ --with-domjudge-user=root
+}
+
+repo-install () {
+    apt-get install $@ -y >/dev/null
+}
+repo-remove () {
+    apt-get remove $@ -y >/dev/null; apt-get autoremove -y 2>/dev/null
+}
+remove_gnuc () {
+    repo-remove gcc g++
+}
+remove_c () {
+    remove_gnuc; repo-remove clang
+}
+
+@test "Default empty configure" {
+    # cleanup from earlier runs
+    remove_c
+    run ./configure
+    assert_failure 1
+    assert_line "checking whether configure should try to set CFLAGS... yes"
+    assert_line "checking whether configure should try to set CXXFLAGS... yes"
+    assert_line "checking whether configure should try to set LDFLAGS... yes"
+    assert_line "checking for gcc... no"
+    assert_line "checking for cc... no"
+    assert_line "checking for cl.exe... no"
+    assert_line "configure: error: in \`/domjudge':"
+    assert_line 'configure: error: no acceptable C compiler found in $PATH'
+    assert_line "See \`config.log' for more details"
+}
+
+compiler_assertions () {
+    run q_configure
+    # Depending on where we run this we might runas wrong user or lack libraries
+    # assert_success
```

Comment:
This looks like a leftover commented line of code.

Reply:
Rewritten to make it clear why this is there.